### ISSUE TITLE
Common instructions in Installation.md

### ISSUE THREAD

**Issue by mattias-p** [dotse/zonemaster-backend#373]
When it's possible to put identical instructions first or last in the OS specific sections, a single common instruction should be used instead.

Pros:
 * Keeping the section in question consistent becomes trivial.
 * It reduces the length of the document and therefore the scrolling.

Cons:
 * Requires branching out to OS specific sections and branching in to common sections.

The branching problem can be solved in one of several ways. My suggestion is using links at the end of the OS specific sections to direct the reader to the common final section.

**Comment by mattias-p** [comment]
@matsduf @sandoche2k Do you guys have opinions on this? I wouldn't have bothered with this, but it has been a contentious topic in the past.

**Comment by sandoche2k** [comment]
I feel that it is better to have the same instructions repeated. This is easy for the user who performs an installation.

**Comment by matsduf** [comment]
Going back and forth is not a good idea, but moving the last step (sending a test command to the RPC API) to the end of the instructions could work if it is obvious that you should go there. @mattias-p  and @sandoche2k, please see my PR #374 with a TOC in the beginning and consistently numbered headlines.

**Comment by mattias-p** [comment]
This issue is no longer relevant after #391 and #393.
